config/internal/common/policy.yaml.tmpl

```diff
@@ -18,6 +18,13 @@ spec:
     # We only allow DSPA components to communicate
     # by bypassing oauth proxy, all external
     # traffic should go through oauth proxy
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              name: openshift-user-workload-monitoring
```

Comment:
please just add this part to from part below after line 28
that would be enough, separate from key would override rest of things.

Reply:
Done, refactored based on this suggestion, pushed to an image and re-deployed. Metrics are scraped correctly:

![Screenshot from 2023-03-23 15-52-52](https://user-images.githubusercontent.com/46318816/227336783-152f8498-bbe9-40c9-ae3b-02899a745ec6.png)